dbt/adapters/athena/connections.py

```diff
@@ -215,6 +215,8 @@ def format(self, operation: str, parameters: Optional[List[str]] = None) -> str:
 
         if operation.upper().startswith(("SELECT", "WITH", "INSERT")):
             escaper = _escape_presto
+        elif operation.upper().startswith("VACUUM"):
+            operation = operation.replace('"', "")
```

Comment:
can you add more operations `OPTIMIZE` should be affected by the same issue.

Reply:
hmm. good question.

Reply:
I can actually confirm that OPTIMIZE is affected by the same issue, I'm using what is described [here](https://github.com/dbt-athena/dbt-athena/pull/191#issuecomment-1489710682) in production. Somehow I missed this part of the codebase.

Reply:
yep. I just tested as well.